### ISSUE TITLE
Textbook Code Generates Error

### ISSUE THREAD

**Issue by Mdcrab02** [Qiskit/qiskit-metapackage#1034]
## Did you find a doc bug or broken link that needs to be fixed?

- What is the file name(s) where you found the bug?

https://qiskit.org/textbook/ch-states/single-qubit-gates.html
pauli_table.py

- What is the current text that needs to be fixed or removed? (You can copy and paste)

Let's see the result of the above circuit. Note: Here we use plot_bloch_multivector() which takes a qubit's statevector instead of the Bloch vector.

`# Let's see the result
backend = Aer.get_backend('statevector_simulator')
out = execute(qc,backend).result().get_statevector()
plot_bloch_multivector(out)`

- Are there other related issues or pull requests about this bug?

I don't believe so.

- The error in the documentation
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-8-ba318b000b18> in <module>
      2 backend = Aer.get_backend('statevector_simulator')
      3 out = execute(qc,backend).result().get_statevector()
----> 4 plot_bloch_multivector(out)

/srv/conda/envs/notebook/lib/python3.7/site-packages/qiskit/util.py in wrapper(*args, **kwargs)
    108             if kwargs:
    109                 _rename_kwargs(func.__name__, kwargs, kwarg_map)
--> 110             return func(*args, **kwargs)
    111         return wrapper
    112     return decorator

/srv/conda/envs/notebook/lib/python3.7/site-packages/qiskit/visualization/state_visualization.py in plot_bloch_multivector(state, title, figsize, rho)
    269                           'matplotlib".')
    270     # Data
--> 271     bloch_data = _bloch_multivector_data(state)
    272     num = len(bloch_data)
    273     width, height = plt.figaspect(1/num)

/srv/conda/envs/notebook/lib/python3.7/site-packages/qiskit/visualization/utils.py in _bloch_multivector_data(state)
    311     for i in range(num):
    312         paulis = PauliTable(np.zeros((3, 2 * (num - 1)), dtype=np.bool)).insert(
--> 313             i, pauli_singles, qubit=True)
    314         bloch_state = [np.real(np.trace(np.dot(mat, rho.data))) for mat in paulis.matrix_iter()]
    315         bloch_data.append(bloch_state)

/srv/conda/envs/notebook/lib/python3.7/site-packages/qiskit/quantum_info/operators/symplectic/pauli_table.py in insert(self, ind, value, qubit)
    323 
    324         # Column insertion
--> 325         if ind > self.num_qubits:
    326             raise QiskitError("Index {} is greater than number of qubits"
    327                               " in the PauliTable ({})".format(ind, self.num_qubits))

TypeError: '>' not supported between instances of 'int' and 'NoneType'

**Comment by Mdcrab02** [comment]
This same issue affects the following code:

```
# Create random 1-qubit state
psi = random_state(1)
# Display it nicely
array_to_latex(psi, pretext="|\\psi\\rangle =")
# Show it on a Bloch sphere
plot_bloch_multivector(psi)
```

on this page:
https://qiskit.org/textbook/ch-algorithms/teleportation.html

**Comment by kdk** [comment]
Thanks for reporting this @mdcrab02 . A fix for this issue was included in yesterday's 0.20.1 release. Feel free to reopen if upgrading to the latest minor release doesn't resolve this issue for you.
